compiler/runtime/TRRelocationTarget.hpp

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
```

Comment:
I think this file should be named 'RelocationTarget.hpp`  for it to fit in with the extensible classes model.

Reply:
This is named TRRelocationRecord to avoid breaking the build of the cmake, because otherwise it would shadow / pick up the wrong file. Thank you for the question!